app/views/pages/dashboard_blocks/_closest_service.html.haml

```diff
@@ -1,4 +1,12 @@
-%button{onclick: "Dashboard.setLocation()"} Update location
+.text-center
+  %button.btn.btn-primary{onclick: "Dashboard.setLocation()"} Update location
 #position_info
 
-#{ data[:closest_service] }
+%table.table.table-hover
+  %tbody
+    - data[:closest_service].first(3).each do |service|
+      %tr
+        %td #{service['name']}
+        %td #{service['vicinity']}
+        %td
+          %button.btn.btn-light{:type => "button"} #{I18n.t('see')}

```

Comment:
What does it do? There isn't linked anywhere.

Reply:
At this moment anywhere:) Our new task in trello will be responsible for that. We can use method get_place_details from google_api module to present details about place. Here you can look at sample response: https://developers.google.com/places/web-service/details

Reply:
ok :)